### PR TITLE
[ICDS Dashboard] Add new CBE fields

### PR DESCRIPTION
**Pull request by Rohit25negi**
Adding three new field to CBE data source:
1. theme of cbe comes from `theme_cbe`
2. Number of targetted beneficiaries: which is evaulated as follows: 

```
"IF #form/cbe_info/theme_cbe = 'third_fourth_month_of_pregnancy' THEN COUNT-SELECTED (pregnant_women_list)

IF #form/cbe_info/theme_cbe = 'annaprasan_diwas' THEN COUNT-SELECTED (children_list)

IF #form/cbe_info/theme_cbe = 'suposhan_diwas' THEN husband_number

IF #form/cbe_info/theme_cbe = 'coming_of_age' THEN COUNT-SELECTED(coming_of_age_list)

IF #form/cbe_info/theme_cbe = 'public_health_message' THEN women+men+community_leaders+ag


```

3. Number of other beneficaries is evaualted as 

```
"IF #form/cbe_info/theme_cbe = 'third_fourth_month_of_pregnancy' THEN others_preganant_women

IF #form/cbe_info/theme_cbe = 'annaprasan_diwas' THEN others_children

IF #form/cbe_info/theme_cbe = 'suposhan_diwas' THEN others_husband

IF #form/cbe_info/theme_cbe = 'coming_of_age' THEN others_coming_of_age

IF #form/cbe_info/theme_cbe = 'public_health_message' THEN others"

```

Ticket: https://app.asana.com/0/1157830937219993/1153173617186880

### CODE REVIEW

**Comment by Rohit25negi** [comment]
@czue  @snopoke  adding you as reviewer since Cal is offline.

File: custom/icds_reports/ucr/data_sources/cbe_form.json [hunk 1]
```diff
@@ -131,6 +131,181 @@
...
+        "column_id": "theme_cbe"
+      },
+      {
+        "column_id": "count_targeted_beneficiaries",
+        "datatype": "integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "root_doc",
```
Comment (czue) [inline custom/icds_reports/ucr/data_sources/cbe_form.json]: Is this necessary? Does it not work if you just replace this with the `property_path` below?

File: custom/icds_reports/ucr/data_sources/cbe_form.json [hunk 2]
```diff
@@ -131,6 +131,181 @@
...
+        }
+      },
+      {
+        "column_id": "count_other_beneficiaries",
+        "datatype": "integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "root_doc",
```
Comment (czue) [inline custom/icds_reports/ucr/data_sources/cbe_form.json]: same comment

**Review by czue — APPROVED** [review]
Thanks for adding the test cases. These indicators are crazy!

File: custom/icds_reports/ucr/data_sources/cbe_form.json (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/ucr/data_sources/cbe_form.json]: Yeah you are right. It works. Thanks for catching it

**Pull request merged**